core/sql/optimizer/ItemColRef.h

```diff
@@ -350,6 +350,14 @@ class ConstValue : public ItemExpr
            NAMemory * outHeap = CmpCommon::statementHeap()
         );
 
+  ConstValue(const NAString& strval,
+           NABoolean isCaseInSensitive,
+           enum CharInfo::CharSet charSet=CharInfo::DefaultCharSet,
+           enum CharInfo::Collation collation=CharInfo::DefaultCollation,
+           enum CharInfo::Coercibility coercibility=CharInfo::COERCIBLE,
+           NAMemory * outHeap = CmpCommon::statementHeap()
+        );
```

Comment:
first, I think isCaseInSensitive is a import attribute,
second, i don't think add this argument to the end of constructor is a good way.
if add this argument in the second position, it will change some code where call it.
so i add a new constructor.

Reply:
Yes, you need to add it as the last argument of the existing constructors.